### ISSUE TITLE
Provider produces error on crd kupoports

### ISSUE THREAD

**Issue by verbotenj** [demeter-run/ext-cardano-kupo#68]
If categories = [], then we can omit the categories parameter out. That won't trigger the error in Kubernetes provider
```
│ Error: Provider produced inconsistent result after apply
│
│ When applying changes to module.ext_cardano_kupo_crds["global"].kubernetes_manifest.customresourcedefinition_kupoports_demeter_run, provider "provider[\"registry.terraform.io/hashicorp/kubernetes\"]" produced an unexpected new value: .object.spec.names.categories: was
│ cty.ListValEmpty(cty.String), but now null.
│
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
```

**Comment by paulobressan** [comment]
@gonzalezzfelipe is it related to this issue #69?

**Comment by paulobressan** [comment]
Hey @verbotenj thanks. @gonzalezzfelipe has already solved this problem in issue #69
